examples/websocket-relay/relay-app/src/components/controls/TaskValue.svelte

```diff
@@ -14,7 +14,7 @@
   async function copy(val: string | null) {
     if (!val) return;
 
-    await clipboardCopy(val);
+    // await clipboardCopy(val);
```

Comment:
@bgins do we want to re-enable this?

Reply:
Yeah, I think so. IIRC, that's what we are using for our copy buttons in the task details.

Reply:
cool, done!